src/test/java/de/tum/in/www1/artemis/service/ExamServiceTest.java

```diff
@@ -58,6 +68,23 @@ public void resetDatabase() {
         database.resetDatabase();
     }
 
+    @Test
+    @WithMockUser(value = "admin", roles = "ADMIN")
+    public void testSetExamExerciseProperties() {
+        StudentParticipation studentParticipation = new StudentParticipation();
+        studentParticipation.setTestRun(true);
+        QuizExercise exercise = new QuizExercise();
+        exercise.setStudentParticipations(Set.of(studentParticipation));
+        studentParticipation.setExercise(exercise);
+        exerciseGroup1.addExercise(exercise);
+        exerciseRepository.save(exercise);
+        studentParticipationRepository.save(studentParticipation);
+
+        examService.setExamExerciseProperties(exam1);
+
+        assertThat(exercise.getTestRunParticipationsExist()).isEqualTo(true);
```

Comment:
It would be great if we could also test the corresponding REST call that should send the information to the client as part of a full server integration test